app/controllers/dashboard/sub_service_requests_controller.rb

```diff
@@ -84,7 +84,7 @@ def show
 
   def update
     if @sub_service_request.update_attributes(sub_service_request_params)
-      @sub_service_request.distribute_surveys if @sub_service_request.is_complete?
+      @sub_service_request.distribute_surveys if (@sub_service_request.status == 'complete' && sub_service_request_params[:status].present?)
```

Comment:
I think instead of checking that the status is only 'complete', this should probably check if the status is in FINISHED_STATUSES

`if (FINISHED_STATUSES.include?(@sub_service_request.status) && sub_service_request_params[:status].present?)`